06week/Checkpoint2.js

```diff
@@ -0,0 +1,105 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = = strNums.split(',').map(function(item) {
```

Comment:
There's an extra "=" here

Also, the nums variable is already an array.  Split() is a string method to transform a string to an array.  You would have been mapping the nums array to return a new array, transforming the string values to numbers

Make sure to use ES6 arrow functions